MatrixSDK/Data/MXRoom.h

```diff
@@ -1291,10 +1291,12 @@ Remove a tag applied on an event of the room
  Returns the read receipts list for an event, excluding the read receipt from the current user.
 
  @param eventId The event Id.
+ @param threadId The thread Id. Use `kMXEventTimelineMain` for the main timeline.
  @param sort YES to sort them from the latest to the oldest.
  @param completion Completion block containing the receipts for an event in a dedicated room.
  */
 - (void)getEventReceipts:(nonnull NSString*)eventId
+                threadId:(nullable NSString*)threadId
```

Comment:
Does it make sense to make this threadId `nonnull` now, as we expect `kMXEventTimelineMain` for the main timeline?